docs/governance/active.md

```diff
@@ -0,0 +1,16 @@
+## Active Governance Proposals
```

Comment:
Change to Approved Governance Proposals

Reply:
Haven't added "Approved" because the page is about having a reference for proposals that have been approved AND are still active during the current Epoch.

For example, when Epoch 1 ends, proposals like HGP-4 would be removed from this page, since the Dev grant have ended.

Reply:
ok makes sense